rootfs/deploy.py

```diff
@@ -26,6 +24,19 @@ def log_output(stream, decode):
         else:
             print(chunk.decode('utf-8'))
 
+
+def stream_error(stream):
+    """Translate docker stream errors into a more digestable format"""
+    error = False
+    for chunk in stream:
+        # error handling requires looking at the response body
+        if 'error' in chunk:
```

Comment:
Could this error-handling logic just be added to the `log_output()` function instead? It seems like we're duplicating some handling of the stream & chunks otherwise.